var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -33,51 +33,73 @@ class Subversion(Package):
     version('1.8.13',    '8065b3698d799507fb72dd7926ed32b6')
     version('1.9.3',     'a92bcfaec4e5038f82c74a7b5bbd2f46')
 
+    variant('python', default=False, description='Provide python bindings')
+    variant('perl', default=False, description='Provide Perl bindings')
+    # variant('ruby', default=False, description='Provide ruby bindings')
+
     depends_on('apr')
     depends_on('apr-util')
     depends_on('zlib')
     depends_on('sqlite')
     depends_on('serf')
+    depends_on('expat')
+    depends_on('swig@:2.99')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+    depends_on('pkg-config', type='build')
 
```

Comment:
Why are these needed?  Normally, tarball downloads come with the `configure` script already made, they don't require one to rerun `autogen.sh`.  What does this have to do with adding Perl + Python bindings?  If this is a "general improvement" of this package, it should go into a different PR, not piggy-backed onto P&P bindings.